examples/07-EMIT/ComputeInterferenceType.py

```diff
@@ -0,0 +1,209 @@
+import sys
+from pyaedt.emit_core.emit_constants import InterfererType, ResultType, TxRxMode
+from pyaedt import Emit
+import pyaedt
+import os
+import pyaedt.generic.constants as consts
+import subprocess
+
+# Check to see which Python libraries have been installed
+reqs = subprocess.check_output([sys.executable, '-m', 'pip', 'freeze'])
+installed_packages = [r.decode().split('==')[0] for r in reqs.split()]
+
+# Install required packages if they are not installed
+def install(package):
+    subprocess.check_call([sys.executable, '-m', 'pip', 'install', package])
+
+# Install any missing libraries
+required_packages = ['plotly']
+for package in required_packages:
+    if package not in installed_packages:
+        install(package)
```

Comment:
Here, I would just add a plain message explaining to the customer/user what we are doing.
This way they don't think we are trying to add any shady packages without notification.

@maxcapodi78 Just to double check, did we ever face a situation where a user cannot install package due to some internal IT restrictions for security?

Reply:
@maxcapodi78 this is quite normal. It may happen for internet restrictions or firewall issues. I agree on adding a text that explains what we are doing and small disclosure about the needs of internet connection

Reply:
Sounds good. Thanks @maxcapodi78

Reply:
Sounds good, added a more specific comment to explain it